.github/workflows/publish-bulk-cdk.yml

```diff
@@ -76,6 +76,16 @@ jobs:
           gradle-distribution-sha-256-sum-warning: false
           arguments: --scan :airbyte-cdk:bulk:bulkCdkBuild
 
+      - name: Integration test Bulk CDK
+        uses: burrunan/gradle-cache-action@v1
+        env:
+          CI: true
+        with:
+          job-id: bulk-cdk-publish
+          concurrent: true
```

Comment:
there are some read-only flags that you can set here, not sure if they're actually helpful

in any case none of this seems wrong